src/mame/video/seta.cpp

```diff
@@ -854,7 +828,8 @@ uint32_t seta_state::screen_update_seta_no_layers(screen_device &screen, bitmap_
 void seta_state::seta_layers_update(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect, int sprite_bank_size, int sprite_setac )
 {
 	int layers_ctrl = -1;
-	int enab_0, enab_1, x_0, x_1=0, y_0, y_1=0;
+	int bank[2], x[2]=0, y[2]=0;
```

Comment:
Official tools complain about:
Compiling src/mame/video/seta.cpp...
../../../../../src/mame/video/seta.cpp: In member function 'void seta_state::set
a_layers_update(screen_device&, bitmap_ind16&, const rectangle&, int, int)':
../../../../../src/mame/video/seta.cpp:831:20: error: array must be initialized
with a brace-enclosed initializer
  int bank[2], x[2]=0, y[2]=0;
                    ^
../../../../../src/mame/video/seta.cpp:831:28: error: array must be initialized
with a brace-enclosed initializer
  int bank[2], x[2]=0, y[2]=0;